clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/restapiv2/requests/SetNodeStatesForClusterRequest.java

```diff
@@ -77,4 +78,10 @@ public SetResponse calculateResult(RemoteClusterControllerTask.Context context)
         // 'true' here means the current state now equals the request's wanted state.
         return new SetResponse("ok", true);
     }
+
+    @Override
+    public boolean isFailed() {
+        // Failure to set a node state is propagated as a 200 with wasModified false.
+        return super.isFailed() || (resultSet && !result.getWasModified());
+    }
```

Comment:
`SetNodeStatesForClusterRequest` does not actually override `RemoteClusterControllerTask.hasVersionAckDependency()` (default `false`) today, but it makes sense to have this here anyway for symmetry.